perl_lib/EPrints/Extras.pm

```diff
@@ -304,7 +304,7 @@ sub render_possible_doi
 
 	$value = "" unless defined $value;
 	if( $value =~ m!^
-			(?:http://(?:dx\.)?doi\.org/)?  # add this again later anyway
+			(?:http(?:s)://(?:dx\.)?doi\.org/)?  # add this again later anyway
```

Comment:
@denics RFC 1808 is obsolete, the better reference is RFC 3986. But in any case, CrossRef recommended we link to the secure site, even if our EPrints site is insecure.